### DISCUSSION TITLE
Support for Default Attributes in Lists and Dictionaries

### DISCUSSION THREAD

**Discussion by Macktireh** [wrabit/django-cotton#160] [Q&A] [answered]
Are you planning to support default attributes for lists and dictionaries, for example:

```html
<c-vars :toast="{ type: 'success', message: 'Hello world!' }" />

<!-- or -->

<c-vars :items="[Dashboard, Properties, Tenants, Tenancies]" />

{% for item in items %}
  <li>
    <c-components.ui.link
      url="app:{{ item|lower }}"
      text="{{ item }}"
      icon="{{ item|lower }}"
    />
  </li>
{% endfor %}
```

Thank you.

**Comment by wrabit** [comment]
Hi @Macktireh yes it’s supported, check the first test case here https://github.com/wrabit/django-cotton/blob/main/django_cotton/tests/test_cvars.py

**Reply by wrabit** [reply, marked as answer]
@Macktireh please ensure you update to the latest release v1.0.9. A bug was spotted in connection to dynamic cvars.

**Reply by Macktireh** [reply]
Great thanks 🙏 it works
